tests/test_minimal/test_spikeextractor_to_nwb.py

```diff
@@ -17,7 +17,6 @@
     get_default_nwbfile_metadata,
 )
 
-from neuroconv import spikeinterface  # testing aliased import
```

Comment:
Just tested locally, `from neuroconv.tools import spikeinterface` should still and be lazy and is good enough for me. Can you try re-adding those aliased tests with that alteration to the import?

Reply:
Sure.